src/components/__test__/Navigation.test.js

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import Navigation from '../Navigation';
+
+import { BrowserRouter } from 'react-router-dom';
+import {
+  render,
+  screen,
+  cleanup,
+  fireEvent,
+  waitFor,
+  within,
+} from '@testing-library/react';
+
+import '@testing-library/jest-dom/extend-expect';
+
+import MockAdapter from 'axios-mock-adapter';
+import AxiosConfig from '../../helpers/axiosConfig';
+import ApiProvider from '../../contexts/ApiProvider';
+import { act } from 'react-dom/test-utils';
+const api = new AxiosConfig();
+
+const mock = new MockAdapter(api.instance, {
+  onNoMatch: 'throwException',
+});
+
+describe('should navigation renders correctly', () => {
+  it('nav logo return text', () => {
+    render(
+      <BrowserRouter>
+        <Navigation />
+      </BrowserRouter>
+    );
+
+    const navLogoText = screen.getByTestId('nav-logo-text');
+    expect(navLogoText.textContent).toBe('Blog-App');

```

Comment:
There is a `toHaveTextContent` matcher. Maybe you can use it?
```suggestion
    expect(navLogoText).toHaveTextContent('Blog-App');

```